tracking_manager/models/models.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2023 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from collections import defaultdict
+
+from odoo import api, models, tools
+
+from ..tools import format_m2m
+
+# To avoid conflict with other module and avoid too long function name
+# specific tracking_manager method are prefixed with _tm
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
```

Comment:
I'm wondering why we could not move all of this in `mail.thread` AbstractModel ? Using this code what's happen on models that do not inherit  from `mail.thread` ?

Reply:
In fact we need to inherit all models.
For example you want to track the change on "sale.order.line" and post this information on the related "sale.order"